packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -332,8 +331,8 @@ class TestConfiguration {
         roleId: roleId,
         appId,
       }
-      const authToken = auth.jwt.sign(authObj, env.JWT_SECRET)
-      const appToken = auth.jwt.sign(app, env.JWT_SECRET)
+      const authToken = auth.jwt.sign(authObj, coreEnv.JWT_SECRETS[0])
```

Comment:
I think it would be preferential if we had a `latest` property and then maybe an `old` array rather than just treating it as an array - comparing it to other environment variables its very odd that this is an array - its not well explained.

Reply:
I was toying with a get latest function as well, I'll add that

Reply:
And similar for api key